src/css/views/studio/text-nodes.css

```
@@ -98,15 +105,131 @@
+    calc(0.985714em + 4.375px)
+  );
+
+  position: relative;
+  box-sizing: border-box;
+  margin-block: var(--ss-studio-text-node-font-size, 1em);
+  padding-block: var(--ss-studio-text-node-code-row);
+  padding-inline-start: var(--size-4-4);
+  padding-inline-end: 0;
+  overflow-wrap: anywhere;
+  white-space: pre-wrap;
+  word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)